src/components/tooltip/tooltip.js

```diff
@@ -442,9 +479,9 @@ function MdTooltipRegistry() {
 
   /**
    * Removes an event handler from the service.
-   * @param  {String} type Type of event handler.
-   * @param  {Function} handler The event handler itself.
-   * @param  {Boolean} useCapture Whether the event handler used event capturing.
+   * @param {string} type Type of event handler.
+   * @param {function} handler The event handler itself.
+   * @param {boolean} useCapture Whether the event handler used event capturing.
```

Comment:
@DevVersion FWIU `string` and `boolean` are non-nullable by definition. I have added the `!` symbol before the Function.